### ISSUE TITLE
since snap package updated today, onlyoffice no longer opens.

### ISSUE THREAD

**Issue by pallebone** [ONLYOFFICE/snap-desktopeditors#75]
Unsure if this error is relevant?

"(DesktopEditors:30551): dbind-WARNING **: 10:58:59.810: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-J7q0wewGuE: No such file or directory
DesktopEditors had been run already
"
